### DISCUSSION TITLE
Unable to open applications in Neurodesk

### DISCUSSION THREAD

**Discussion by ahnetzley** [neurodesk/neurodesk.github.io-hugo#438] [Q&A]
I have used Neurodesk play to see if this could be a solution to my neuroimaging problems, and while I was unable to download and save any of the data created, it seemed like it would serve my purposes so I downloaded and installed Neurodesk for my local computer (Mac). 
I am trying to take a look at my data with FSLeyes, but Neurodesk is unable to open anything. When I try to open FSL, the terminal pops up, it gives a fatal error and immediately closes. I cannot even read what the error is because it closes before I can see. Does anyone know a work around for opening FSL, or AFNI, or anything in Neurodesk?

**Comment by stebo85** [comment]
Is it working for you now? Did you run this on Mac with apple silicon?

**Reply by ahnetzley** [reply]
Definitely not working. It seems that the issue is that M3 Mac is not supported so no matter what I do, it will never work until someone smarter than me figures it out.

**Reply by stebo85** [reply]
Yes GUI applications currently do not work on apple silicon, but we are working on a solution for that.

**Reply by ahnetzley** [reply]
It was doing really well until I tried to launch the FSL terminal. Seems that even with Docker and Rosetta it still tries to open using ARM64 instead of AMD64. I'm trying it on an old windows machine now, but its very slow.

**Reply by stebo85** [reply]
Yes, Rosetta support is currently broken in docker- we are waiting for a fix. You can try podman as a workaround - but gui applications will still not work well. 

An older x86 system will work as a workaround for now.

**Reply by ahnetzley** [reply]
I found a temporary work around that will serve my purposes. Neurodesktop play, in the browser, works perfectly fine except that I cannot drag and drop zipped files from Neurodesk to my PC, however, I was able to log into dropbox via firefox and upload the zipped file there, and then download from firefox onto my hard drive.

**Reply by stebo85** [reply]
Ok, you can download files either through the desktop or through the jupyterlab interface: https://www.neurodesk.org/docs/getting-started/neurodesktop/storage/#downloading-files
